woocommerce/woocommerce-android
# [M6 Orders With Coupons] Expandable product cards in order creation - part 2

## Expandable product cards in order creation - part 2
This is part 2 of the following PRs:
* Part 1: https://github.com/woocommerce/woocommerce-android/pull/9851 
* **Part 2: https://github.com/woocommerce/woocommerce-android/pull/9852**
* Part 3: https://github.com/woocommerce/woocommerce-android/pull/10018

#### ⚠️ Do not merge

### Description

This PR adds a basic expandable product cards UI, as per design: FTOWxedKUyDbBdOrxRfMyI-fi-14%3A9617.
The remaining, elements of the cards and various UI improvements are done in the next PR: https://github.com/woocommerce/woocommerce-android/pull/10018

### Testing instructions

Create a new order and add new products. Observe the new expandable product cards are rendered instead of the old ones.

### Images/gif


https://github.com/woocommerce/woocommerce-android/assets/4527432/fdfa194f-e3de-4f92-a9eb-5b45135f346d
